packages/connect/src/data/coinInfo.ts

```diff
@@ -52,7 +41,7 @@ export const getEthereumNetwork = (pathOrName: number[] | string) => {
 };
 
 export const getMiscNetwork = (pathOrName: number[] | string) => {
-    const networks = cloneCoinInfo(miscNetworks);
+    const networks: MiscNetworkInfo[] = window.structuredClone(miscNetworks);
```

Comment:
Did you checked output in /lib that this is really correctly transpiled and polyfilled?

Reply:
even if it polyfilled correctly you would still need to persuade me that it is good :D

Reply:
Why not? It is better to use build-in api that something DIY.

Reply:
LOL, it's the JSON way again x)

```
function cloneCoinInfo(info) {
    const jsonString = JSON.stringify(info);
    if (jsonString === undefined) {
        return info;
    }
    return JSON.parse(jsonString);
}
```

Reply:
Nope, it stays as  just `structuredClone()` :(

Reply:
from the discussion about backporting this to Node v16 (currently available from v17) https://github.com/nodejs/node/issues/40756#issuecomment-1032530509 🤷

Reply:
maybe it's better to wait until we upgrade to a higher version of Node to start using this?

Reply:
It is also running in the browser in iframe right? https://caniuse.com/mdn-api_structuredclone It is not supported in all Suite supported browsers. Connect should be supported in even more browsers than Suite.

Reply:
I created a separate util after all and only use it if available as discussed with @Nodonisko  c21c17866a76fea7094329f79dc0bb67ddb106bb

Reply:
I think in Suite webpack should polyfill anyway if target is correctly set. If not I not I will have re-study how these polyfiling works...